### PR TITLE
Added client folder

### PR DESCRIPTION
**Pull request by StephenEsser**
Checking in the client folder

### CODE REVIEW

File: Rails/client/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,10311 @@
+{
```
Comment (emilyrohrbough) [inline Rails/client/package-lock.json]: Do you want to check this in?

**Comment by emilyrohrbough** [comment]
Did you want to check in Rails/client/aggregated-translations/.gitkeep?

**Comment by mjhenkes** [comment]
@emilyrohrbough yes actually, without that folder we get an error when webpacking

File: Rails/client/package-lock.json (hunk 1 above)
Comment (StephenEsser) [reply Rails/client/package-lock.json]: Yes, the application is the last consumer in the dependency chain so checking in the package lock is safe

**Pull request merged**